src/modules/partners/partner_entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Url } from 'url';
+import { Query } from '../../helpers/types';
+
+export namespace Partner {
+  export interface Struct {
+    id?: string
+    name: string
+    category_id?: number
+    logo?: Url
+    verified_at?: Date
+    total_village: number
+    deleted_at?: Date
+    updated_at?: Date
+    created_at?: Date
```

Comment:
if I'm not mistaken, we have to put our partner's website on the landing page, right? Or will this button redirect us to another page of DesaDigital?
![image](https://user-images.githubusercontent.com/52171676/136719297-9d16637d-d75f-46b5-8a5c-fce3bff3b439.png)

Reply:
updated 16659de